plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreProcedure.java

```diff
@@ -828,7 +828,8 @@ public List<PostgreDependency> getDependencies(DBRProgressMonitor monitor) throw
 
     @Override
     public boolean supportsObjectDefinitionOption(String option) {
-        return DBPScriptObject.OPTION_INCLUDE_COMMENTS.equals(option) || DBPScriptObject.OPTION_INCLUDE_PERMISSIONS.equals(option);
+        return DBPScriptObject.OPTION_INCLUDE_COMMENTS.equals(option) || DBPScriptObject.OPTION_INCLUDE_PERMISSIONS.equals(option) 
+                || DBPScriptObject.OPTION_CAST_PARAMS.equals(option);
```

Comment:
We use 4 spaces for continuation indent:

```suggestion
        return DBPScriptObject.OPTION_INCLUDE_COMMENTS.equals(option)
            || DBPScriptObject.OPTION_INCLUDE_PERMISSIONS.equals(option) 
            || DBPScriptObject.OPTION_CAST_PARAMS.equals(option);
```